motion/motion.go

```diff
@@ -81,14 +80,23 @@ func (d *motionDetector) pixelsChanged(frame *lepton3.Frame) (bool, int) {
 
 	if !d.firstDiff {
 		d.firstDiff = true
-		return false, NO_DATA
+		return false, 0
+	}
+
+	if isAffectedByFFC(frame) {
```

Comment:
This will mean that videos always stop during the FFC since it takes about 10s and videos will stop recording after 10s of no movement detected.

Reply:
Yep, will work on this as dicussed.